androidClient/app/src/main/java/com/macbitsgoa/comrades/HomeActivity.java

```diff
@@ -69,8 +70,8 @@
                     homeFragment = HomeFragment.newInstance();
                 }
                 fragmentManager.beginTransaction().replace(R.id.container_fragment,
-                        homeFragment, TAG_HOME_FRAG).addToBackStack(null).commit();
-                break;
+                        homeFragment, "HomeFragment").addToBackStack(null).commit();
```

Comment:
keep the constant, that way someday we won't misspell it somewhere

Reply:
didn't noticed that